kwsCheckMemberFunctions.cxx

```diff
@@ -26,7 +26,7 @@ bool Parser::CheckMemberFunctions(const char* regEx,unsigned long maxLength)
     {
     m_TestsDone[MEMBERFUNCTION_LENGTH] = true;
     m_TestsDescription[MEMBERFUNCTION_LENGTH] = "Member functions must not exceed: ";
-    m_TestsDescription[MEMBERFUNCTION_LENGTH] += maxLength;
+    m_TestsDescription[MEMBERFUNCTION_LENGTH] += std::string(maxLength);
```

Comment:
Did you mean `std::to_string`?

Reply:
yes. from what i understand the intend was to append maxLength to the m_Test Description but the problem was it was casting maxLenght to char of its ASCII code. And it was giving -Wimplicit-int-conversion

Reply:
This cannot be right, as it doesn't even compile: `C:\Dev\KWStyle\kwsCheckMemberFunctions.cxx(29,61): error C2440: '<function-style-cast>': cannot convert from 'unsigned long' to 'std::string'`

Reply:
`std::to_string` is the right thing here. This snippet:
```cpp
#include <string>
#include <iostream>

int
main()
{
  std::string   description = "Member functions must not exceed: ";
  unsigned long maxLength = 100;
  description += maxLength;
  description += " lines";

  std::cout << description << std::endl;
}
```
produces:
`Member functions must not exceed: d lines`.

Reply:
you are right. i just saw it now. i close this pull request

Reply:
Do you want to make a follow-up PR?

Reply:
yes